messaging/src/main/java/org/axonframework/deadline/jobrunr/JobRunrDeadlineManager.java

```diff
@@ -339,6 +344,19 @@ public Builder spanFactory(@Nonnull DeadlineManagerSpanFactory spanFactory) {
             return this;
         }
 
+        /**
+         * Sets the {@link MessageNameResolver} used to resolve the {@link QualifiedName} when publishing {@link EventMessage EventMessages}.
```

Comment:
Darn, not sure if I suggested this wrongly now, but regardless, the JavaDoc should be about `DeadlinMessages`, not `EventMessages` :-[